include/igl/copyleft/comiso/nrosy.cpp

```diff
@@ -803,8 +805,16 @@ Eigen::VectorXd igl::copyleft::comiso::NRosyField::angleDefect()
     {
       Eigen::VectorXd a = V.row(F(i,(j+1)%3)) - V.row(F(i,j));
       Eigen::VectorXd b = V.row(F(i,(j+2)%3)) - V.row(F(i,j));
-      double t = a.transpose()*b;
-      t /= (a.norm() * b.norm());
+      double t = a.transpose() * b;
+      double norm_prod = a.norm() * b.norm();
+      if (norm_prod > 0.)
+        t /= norm_prod;
+      else
+        throw std::runtime_error ("Error in 'igl::copyleft::comiso::NRosyField::angleDefect': division by zero!");
```

Comment:
Sorry this one also should be `std::runtime_error(...`. Thanks!

Reply:
Sorry, my bad. Anyway fixed now.